x-pack/plugins/apm/public/components/shared/UIComponents.js

```diff
@@ -40,6 +47,12 @@ export const HeaderSmall = styled.h3`
   ${props => props.css};
 `;
 
+export const HeaderXSmall = styled.h4`
+  margin: ${px(units.plus)} 0;
+  font-size: ${fontSize};
+  ${props => props.css};
```

Comment:
if you wondered about the css prop, it's a neat/hacky trick for allowing real css: https://github.com/sqren/kibana/blob/3988e3639f7d354dc9b5578daf56b4d93fc00f53/x-pack/plugins/apm/public/components/app/TransactionDetails/Distribution/view.js#L68-L73

Reply:
Uhhh, that's very nice 👍

Reply:
yeah, it let's us have some standard headers, that can still be modified locally.